hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -132,17 +163,100 @@ private HederaAccountCustomizer asCustomizer(CryptoCreateTransactionBody op) {
         long consensusTime = txnCtx.consensusTime().getEpochSecond();
         long expiry = consensusTime + autoRenewPeriod;
 
-        /* Note that {@code this.validate(TransactionBody)} will have rejected any txn with an invalid key. */
-        final JKey key = asFcKeyUnchecked(op.getKey());
-        HederaAccountCustomizer customizer =
-                new HederaAccountCustomizer()
-                        .key(key)
-                        .memo(op.getMemo())
-                        .expiry(expiry)
-                        .autoRenewPeriod(autoRenewPeriod)
-                        .isReceiverSigRequired(op.getReceiverSigRequired())
-                        .maxAutomaticAssociations(op.getMaxAutomaticTokenAssociations())
-                        .isDeclinedReward(op.getDeclineReward());
+        var customizer = new HederaAccountCustomizer();
+        if (!op.getAlias().isEmpty() && !op.hasKey()) {
```

Comment:
nit: Extract this into a `boolean validAlias = !op.getAlias().isEmpty()` for easy reading in all the scenarios below
```suggestion
        // alias with no key case
        if (validAlias && !op.hasKey()) {
```